src/rex.core/src/rex/core/testing.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2020, Prometheus Research, LLC
+#
+
+
+class RexTestSuite:
+    """
+    A ``pytest``-compatible test suite class that will automatically activate
+    a Rex application for the duration of the tests defined as part of the
+    class.
+
+    Requires that the ``create_app()`` method be implemented.
+    """
+
+    #: The currently-active Rex application.
+    rex = None
+
+    @classmethod
+    def create_app(cls):
+        """
+        Creates and returns the Rex application to activate for the tests
+        executed in this suite.
+
+        Must be implemented in concrete classes.
+
+        :rtype: rex.core.Rex
+        """
+
+        raise NotImplementedError(
+            f'{cls.__name__}.create_app() must be implemented'
+        )
+
+    @classmethod
+    def setup_class(cls):
+        cls.rex = cls.create_app()
+        cls.rex.on()
+
+    @classmethod
+    def teardown_class(cls):
+        cls.rex.off()
+        cls.rex = None
+
+
+class any:
```

Comment:
Hm... I think I don't understand how this is going to be used. Isn't `assert x == any()` will be pass for any `x`, what's the assertion is doing then?

Reply:
The example is perhaps a bit simplistic, but this comes in handy when you're asserting an entire dictionary at once. E.g.:

```
assert something = {
  'foo': 1,
  'bar': any(),
  'baz': 'red',
}
```

This lets you assert the structure of the entire dictionary more clearly than X separate statements. You check that the dictionary contains exactly the keys you expect, but if there's one key whose value you don't care about what it specifically is, you can use `any()` as the placeholder.

Reply:
Oh, that's useful!